package.json

```
@@ -4,7 +4,14 @@
   "description": "",
   "main": "index.js",
   "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
+    "client": "yarn --cwd packages/client start",
+    "build": "yarn --cwd packages/client build",
+    "server": "yarn --cwd packages/server/ dev",
+    "parallel": "npm-run-all --parallel client build server",
```

Wait why'd we switch from `concurently` to `npm-run-all` whatever the case this can be shortened. It doesn't make sense on `parallel` to run the build since that is _dev_, we still need a start command:
```suggestion
    "client": "yarn --cwd packages/client start",
    "build": "yarn --cwd packages/client build",
    "server": "yarn --cwd packages/server dev",
    "dev": "run-p client server",
    "start": "yarn --cwd packages/server start"
```

We were using something besides `cwd` is there a reason that was changed?

---

Thanks, indifferent for the suggestions. Yeah, I went ahead and went with npm run all per the instructions. I will go ahead and get the requested changes committed. Appreciate the help with this and taking it easy on a first-timer.